admin/client/src/components/form/styles.scss

```diff
@@ -0,0 +1,4 @@
+.form-component {
```

Comment:
I don't think we should suffix BEM style blocks with `-component` - I've also renamed the React class from `FormComponent` to `Form` through https://github.com/silverstripe/silverstripe-framework/pull/5372